stacks/core/README.md

```diff
@@ -19,7 +19,20 @@ The `core` stack uses multiple compose files to setup the services and reach the
 
 #### Development
 
-When the stack is copied to a specified workspace through the CLI `init` operation, the following command must be executed to deploy the services:
+After using the `init` command as described [here](../../README.md#Development), the stack files will be copied to `<path>/stack` directory (`path` is the folder specified in the `init` execution), be sure to enter there. The following files should be shown when exploring the `stack` directory
```

Comment:
Hadn't commented before but great job on this paragraph, this is somewhat complex information being conveyed in a clear and succinct manner.